metadata-service/auth-impl/src/main/java/com/datahub/authorization/AuthorizerChain.java

```diff
@@ -126,11 +126,16 @@ private AuthorizedActors mergeAuthorizedActors(@Nullable AuthorizedActors origin
       mergedGroups = new ArrayList<>(groups);
     }
 
+    Set<Urn> roles = new HashSet<>(original.getRoles());
```

Comment:
This method seems like dead code? I don't see any references to it outside of this class and tracing back to Dexter's PR I don't see any actual usages of it. To get the actors authorized for something we go through different code unless I'm missing something?

Reply:
I am not super familiar with the code. 
My understanding is that this merging logic is for when we have multiple chained authorizers. 
Since the interface [extended a plugin interface](https://github.com/datahub-project/datahub/blob/master/metadata-auth/auth-api/src/main/java/com/datahub/plugins/auth/authorization/Authorizer.java#L17), this was something we supported for the community.